metaflow/parameters.py

```diff
@@ -40,7 +40,7 @@
 
 
 class JSONTypeClass(click.ParamType):
-    name = "JSON"
+    name = __name__ = "JSON"
```

Comment:
Needed to supply the type information in a standardized way across all types.  The rest of the types are builtins which already have this field.